<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>675</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1010</width>
    <height>675</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1010</width>
    <height>675</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QVBoxLayout" name="vertical_layout_trees">
      <item>
       <layout class="QHBoxLayout" name="tree_filter_layout">
        <item>
         <widget class="QLineEdit" name="tree_filter_input">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="placeholderText">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tree_filter_reset_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="platform_tabs">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>410</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>410</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="wii_tab">
         <attribute name="title">
          <string>Wii</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTreeView" name="wii_title_tree"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="vwii_tab">
         <attribute name="title">
          <string>vWii</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTreeView" name="vwii_title_tree"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="dsi_tab">
         <attribute name="title">
          <string>DSi</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTreeView" name="dsi_title_tree"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="vertical_layout_controls">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontal_layout_title_entry">
        <item>
         <widget class="QLineEdit" name="tid_entry">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Title ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>v</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="version_entry">
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Version</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Console:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="console_select_dropdown">
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>0</height>
           </size>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string/>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="download_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Start Download</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="script_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Run Script</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>General Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="WrapCheckboxWidget" name="pack_archive_checkbox" native="true"/>
          </item>
          <item>
           <widget class="QLineEdit" name="archive_file_entry">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>File Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="WrapCheckboxWidget" name="keep_enc_checkbox" native="true"/>
          </item>
          <item>
           <widget class="WrapCheckboxWidget" name="create_dec_checkbox" native="true"/>
          </item>
          <item>
           <widget class="WrapCheckboxWidget" name="use_local_checkbox" native="true"/>
          </item>
          <item>
           <widget class="WrapCheckboxWidget" name="use_wiiu_nus_checkbox" native="true"/>
          </item>
          <item>
           <widget class="WrapCheckboxWidget" name="patch_ios_checkbox" native="true">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>200</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>vWii Title Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="WrapCheckboxWidget" name="pack_vwii_mode_checkbox" native="true">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>App Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="WrapCheckboxWidget" name="auto_update_checkbox" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="WrapCheckboxWidget" name="custom_out_dir_checkbox" native="true"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="custom_out_dir_entry_row">
            <item>
             <widget class="QLineEdit" name="custom_out_dir_entry">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="placeholderText">
               <string>Output Path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="custom_out_dir_btn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Select...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>200</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="log_text_browser">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>222</height>
         </size>
        </property>
        <property name="markdown">
         <string/>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progress_bar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_about_qt"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_options">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menu_options_language">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="action_language_system"/>
     <addaction name="action_language_english"/>
     <addaction name="action_language_spanish"/>
     <addaction name="action_language_german"/>
     <addaction name="action_language_french"/>
     <addaction name="action_language_italian"/>
     <addaction name="action_language_norwegian"/>
     <addaction name="action_language_romanian"/>
     <addaction name="action_language_korean"/>
    </widget>
    <addaction name="menu_options_language"/>
   </widget>
   <addaction name="menu_options"/>
   <addaction name="menu_help"/>
  </widget>
  <action name="action_about">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>About NUSGet</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="action_about_qt">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="action_language_system">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>System (Default)</string>
   </property>
  </action>
  <action name="action_language_english">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="action_language_spanish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Español</string>
   </property>
  </action>
  <action name="action_language_german">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Deutsch</string>
   </property>
  </action>
  <action name="action_language_french">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Français</string>
   </property>
  </action>
  <action name="action_language_italian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Italiano</string>
   </property>
  </action>
  <action name="action_language_norwegian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Norsk</string>
   </property>
  </action>
  <action name="action_language_romanian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Română</string>
   </property>
  </action>
  <action name="action_language_korean">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>한국어</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WrapCheckboxWidget</class>
   <extends>QWidget</extends>
   <header>qt/py/ui_WrapCheckboxWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
